### PR TITLE
Feature/services status cluster

### LINKED ISSUE #714: Show services status for all servers in active/active cluster
/admin/status#service is showing only 1 server.

Either we should make it so we can select the server or merge the servers into different columns

It may be worth it to have a 'Cluster status' page where you see the services and communication status of the servers

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Describe what the new code is used for.
ie: Allow PacketFence to trigger the coffee machine on wireless EAP 802.1X connection to brew a fresh espresso.
# Impacts

Show service status for all cluster members
# Issue

fixes #714 
# Delete branch after merge

YES
# NEWS file entries
## New Features
- Added page to visualize the status of the services on all cluster members

### CODE REVIEW

**Comment by julsemaan** [comment]
Please review

File: lib/pf/api.pm [hunk 1]
```diff
@@ -1313,6 +1314,21 @@ sub handle_accounting_metadata : Public {
 
 }
 
+sub services_status : Public {
```
Comment (jrouzierinverse) [inline lib/pf/api.pm:1323]: documentation

**Comment by julsemaan** [comment]
re-review

File: lib/pf/api.pm [hunk 2]
```diff
@@ -1313,6 +1314,27 @@ sub handle_accounting_metadata : Public {
...
+
+=cut
+
+sub services_status : Public {
+    my ($class, $services) = @_;
+    my @managers = pf::services::getManagers($services);
+
+    my $statuses = {};
+    foreach my $manager (@managers){
+        my $isManaged = $manager->isManaged;
```
Comment (jrouzierinverse) [inline lib/pf/api.pm]: Unused variable

File: lib/pf/api.pm [hunk 3]
```diff
@@ -1313,6 +1314,27 @@ sub handle_accounting_metadata : Public {
...
+        my $status = $manager->status;
```
Comment (jrouzierinverse) [inline lib/pf/api.pm]: This should be done within the if below.
Since we do not use any where else

**Comment by julsemaan** [comment]
adjustments done, good for another review

**Pull request merged**